user/id.go

```diff
@@ -0,0 +1,34 @@
+package user
+
+import (
+	"fmt"
+
+	"golang.org/x/net/context"
+)
+
+// TODO(jml): typedef for userid, maybe even put in a weaveworks library, so
+// that there's a shared language around the multiple ways of identifying
+// entitiies.
+
+// UserIDContextKey is the key used in contexts to find the userid
+const userIDContextKey = "CortexUserID" // TODO dedupe with storage/local
+
+// UserIDHeaderName is a legacy from scope as a service.
+const UserIDHeaderName = "X-Scope-OrgID"
+
+// LowerUserIDHeaderName as gRPC / HTTP2.0 headers are lowercased.
+const LowerUserIDHeaderName = "x-scope-orgid"
```

Comment:
Really uncomfortable calling these `UserIDHeaderName` since we already have a `userIDHeader` constant set to "X-Scope-UserID": https://github.com/weaveworks/service/blob/25bba1e6bdc6cc1ff7a0befa53362571dfea5021/authfe/main.go#L22

Please rename to OrgID